frontend/ui-picasso/app/pages/crowdloan-rewards/index.tsx

```diff
@@ -36,6 +36,8 @@ const CrowdloanRewards: NextPage = () => {
         ? router.push("crowdloan-rewards/stablecoin")
         : router.push("crowdloan-rewards/ksm");
     }
+    // Only to be called on page load therefore we can omit dependencies.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
The same - We could remove the dependency list instead

Reply:
i can remove dependancy list for the ones that only run at first